bindings/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+mod bindings {
+    wit_bindgen::generate!({
+        path: "../wit",
+        world: "reactor",
+        std_feature,
+        // The generated definition of command will pull in std, so we are defining it
+        // manually below instead
+        skip: ["command", "get-preopens", "get-environment"],
+    });
+}
+
+pub use bindings::*;
```

Comment:
lets just run the macro at the top level rather than in a mod and pub use it all